css/requirements.txt

```diff
@@ -1,5 +1,5 @@
-six==1.10.0
+six==1.11.0
 Template-Python==0.1.post1
-html5lib==0.9999999
-lxml==3.7.3
-mercurial==4.1
+html5lib==1.0.1
```

Comment:
The html5lib update here is the cause of the allow_failures CSS build job failure (given API changes). We probably don't want this automatically updated.